README.md

```diff
@@ -64,7 +64,6 @@
 - [ ] **[The user can create a new course by himself]**
```

Comment:
- [ ] OPTIONAL: I can't find a way to logout.  I suggest you add a way to logout from the app, so multiple users can use it on the same machine.  

_Tip: As of now, the only way to logout is to remove localStorage.  You could use this and implement a button that deletes localStorage and the user will be logged out. 👍_